### PR TITLE
[HMA-3244] Uplift getPreferences response to match future structure

### PR DESCRIPTION
**Pull request by mrolfe88**

### CODE REVIEW

**Comment by mrolfe88** [comment]
test this please

File: app/uk/gov/hmrc/customerprofile/connector/preferencesFrontendConnector.scala [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+//TODO: Add this call when endpoint details are known
+//  def getStatus(
+//                    )(implicit headerCarrier: HeaderCarrier,
+//                      ex:                     ExecutionContext
+//                    ): Future[Option[StatusWithUrl]] =
+//    withCircuitBreaker(http.GET[Option[StatusWithUrl]](url(s"/paperless/status")))
+//      .recover {
+//        case _:        NotFoundException                                            => None
+//      }
+
```
Comment (georgeherby) [inline app/uk/gov/hmrc/customerprofile/connector/preferencesFrontendConnector.scala:46]: ?????

**Review by georgeherby — CHANGES_REQUESTED** [review]

**Review by georgeherby — APPROVED** [review]

**Comment by mrolfe88** [comment]
test this please

**Comment by mrolfe88** [comment]
test this please

**Pull request merged**